packages/block-library/src/post-hierarchical-terms/index.php

```diff
@@ -19,6 +19,9 @@ function render_block_core_post_hierarchical_terms( $attributes, $content, $bloc
 	}
 
 	$post_hierarchical_terms = get_the_terms( $block->context['postId'], $attributes['term'] );
+	if ( is_wp_error( $post_hierarchical_terms ) ) {
+		return '';
+	}
 	if ( empty( $post_hierarchical_terms ) ) {
```

Comment:
This new condition could be combined with `empty` check.

Reply:
Agreed. What do the coding guidelines say?

Reply:
I'm not aware of coding styles that cover it. I can merge as is if you prefer.

Reply:
Yes please. Then I can continue working on PRs for other issues.